src/model.py

```diff
@@ -320,6 +320,10 @@ def to_misp(self) -> List[MISPObject]:
             obj = MISPObject("email", standalone=False)
             obj.add_attribute("from", email)
             to_return.append(obj)
+
+        # filter out objects without any attributes
+        to_return = list(filter(lambda x: bool(x.attributes), to_return))
```

Comment:
i prefer list comprehension but meh, looks good.

When does it even happen? Objects without attributes I mean. Some weird network locations?

Reply:
y